src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.model;
+
+public enum Operation {
+        BALANCE,
+        SUPPLY,
+        PURCHASE,
+        RETURN;
```

Comment:
Should be something like this
```suggestion
    BALANCE("b"),
    SUPPLY("s"),
    PURCHASE("p"),
    RETURN("r");

    public final String shortValue;

    Operation(String shortValue) {
        this.shortValue = shortValue;
    }

    public static Operation byShortValue(String shortValue) {
        for (Operation operation : values()) {
            if (operation.shortValue.equals(shortValue)) {
                return operation;
            }
        }
        return null;
    }
```